.github/workflows/pr-closed.yaml

```diff
@@ -0,0 +1,17 @@
+name: PR Closed
+
+on:
+  pull_request_target:
+    branches:
+      - main
+    types:
+      - closed
+
+jobs:
+  cherry_pick_job:
+    permissions:
+      pull-requests: write
+      contents: write
+    if: github.event.pull_request.merged == true
+    secrets: inherit
+    uses: trustification/release-tools/.github/workflows/cherry-pick.yaml@main
```

Comment:
Can we rename that too to `backport` to keep it aligned?

Reply:
Done. The workflow is not named `backport.yaml` and the name of it is `name: backport`